supplementary_style_guide/introduction/whats-new.adoc

```diff
@@ -0,0 +1,36 @@
+[[whats-new]]
+= What's new
+
+////
+Instructions:
+- Wait until a month is over to include its entries
+- Only include updates that are significant, not just cleanup, formatting, or typo fixes.
+////
+
+Review the history of significant updates to the _Red Hat supplementary style guide for product documentation_.
+
+* xref:year-2023[2023]
+* xref:year-archive[2020-2022]
+
+[discrete]
+[[year-2023]]
+== 2023
+
+[discrete]
+[[january-2023]]
+=== January 2023
+
+* Removed "through" entry in favor of following the _IBM Style_ guidance on number ranges
```

Comment:
```suggestion
* Removed "through" entry in favor of following the _IBM Style_ guidance on number ranges.
```

Reply:
@mportman12 I do already link to all of the entries (or am I missing something)? Only "through" is not linked, because it was removed (so it doesn't have a place to link to anymore).